docs/guides/automation/cron_jobs_howto.md

```diff
@@ -68,18 +68,18 @@ This translates to the following:
 
 If you want to run scripts on a workstation or notebook in the `cron` "dot" directories, nothing special is needed. Just copy your script file into the directory in question, and make sure it is executable. Here are the directories:
 
-* `/etc/cron.hourly` - Scripts placed here will run one minute past the hour every hour. (this is run by `cron` regardless of whether `anacron` is installed or not)
-* `/etc/cron.daily` - Scripts placed here will run every day. `anacron` adjusts the timing of these. (see tip)
-* `/etc/cron.weekly` - Scripts placed here will run every 7 days, based on the calendar day of the last run time. (see tip)
-* `/etc/cron.monthly` - Scripts placed here will run monthly based on the calendar day of the last run time. (see tip)
+* `/etc/cron.hourly` - Scripts placed here will run one minute past the hour every hour (this is run by `cron` regardless of whether `anacron` is installed or not).
+* `/etc/cron.daily` - Scripts placed here will run every day. `anacron` adjusts the timing of these (see tip).
+* `/etc/cron.weekly` - Scripts placed here will run every 7 days, based on the calendar day of the last run time (see tip).
+* `/etc/cron.monthly` - Scripts placed here will run monthly based on the calendar day of the last run time (see tip).
 
-!!! tip
+!!! tip "Tip"
```

Comment:
@SergeCroise The changes in the admonition capitalization is totally unnecessary. Please see [this article](https://docs.rockylinux.org/guides/contribute/rockydocs_formatting/) The admonition capitalization happens automatically unless you are doing a custom admonition. There's no need in changing a standard admonition to a custom to capitalize the way the admonition shows in the source.

Reply:
The other changes here are good, but not the admonition changes.

Reply:
Actually, the periods in the bullet points in the section above are better removed entirely, since they are "bullet points"... in my opinion.

Reply:
Hello @sspencerwire ,
I reverted the admonition changes and removed the periods.
Please review!